source/main/modules/fsal/util/sort.ts

```diff
@@ -72,5 +72,8 @@ export default function (
   const ret: Array<MDFileDescriptor | DirDescriptor | CodeFileDescriptor> = []
 
   // Return sorted array files -> directories
+  if (global.config.get('sortFoldersFirst')) {
+    return ret.concat(d).concat(f)
+  }
```

Comment:
I would make this an else here so it's clear that one of these results will happen; I think it makes it cleaner!